Balance/Login/LoginView.swift

```diff
@@ -19,16 +19,17 @@ struct LoginView: View {
     @AppStorage(StorageKeys.onboardingFlowComplete)
     var completedOnboardingFlow = false
     @EnvironmentObject var firebaseAccountConfiguration: FirebaseAccountConfiguration<FHIR>
-    @Binding private var onboardingSteps: [OnboardingFlow.Step]
-    
+    @Binding var onboardingSteps: [OnboardingFlow.Step]
     @State private var emailAddress: String = ""
     @State private var password: String = ""
-    
+    @State private var alertMessage: String = ""
+    @State private var showingAlert = false
+
     var body: some View {
         Group {
             if !authModel.isLoggedIn {
-                signInView
-                    .adaptsToKeyboard()
+                    signInView
```

Comment:
Gonza, not sure this will be a problem but just flagging you this variable name `signInView` since I saw you did some sort of renaming from `SignIn` to `SignUp`? 
@gperisset